seed/csharp-sdk/examples/src/SeedExamples/Service/Requests/GetMetadataRequest.cs

```diff
@@ -4,7 +4,7 @@ public record GetMetadataRequest
 {
     public bool? Shallow { get; set; }
 
-    public string? Tag { get; set; }
+    public IEnumerable<string?> Tag { get; set; } = new List<string?>();
```

Comment:
is there any reason to keep the `string`s optional?

Reply:
Great catch - dropping the optional would be consistent with both [TypeScript](https://github.com/fern-api/fern/blob/173e2d44467772d8cddcd2726839b29da43abf4f/seed/ts-sdk/examples/examples-with-api-reference/src/api/resources/service/client/requests/GetMetadataRequest.ts#L15) and [Python](https://github.com/fern-api/fern/blob/3e5494da22fc7b77233ba8d36516fbc73a63128f/seed/python-sdk/examples/no-custom-config/src/seed/service/client.py#L155). I'll make that change here.